### ISSUE TITLE
How Can One Map a Construct to the Process it Belongs to?

### ISSUE THREAD

**Issue by VincenzoFerme** [benchflow/data-transformers#92]
Currently it seems there is no way to map a construct from the `construct` tables, to the process it belongs to. Why? It should be possible to have this map, using the ids of the processes table as well.

**Comment by Cerfoglg** [comment]
@VincenzoFerme Isn't that what the source_process_instance_id columns in the construct table is for?

**Comment by VincenzoFerme** [comment]
Ok, but how can I do this mapping if in the tables containing the construct metrics you do not store the construct id?

**Comment by Cerfoglg** [comment]
@VincenzoFerme But the metrics are computed over multiple constructs with different ids, the metrics are computed over construct names and types if defined.

**Comment by VincenzoFerme** [comment]
Right, that was to aggregate construct info with more flexibility than only for single process.

Evaluate a possible improvement:
- Collect all the `distinct` process definitions to which the constructs belong to and store them
- Compute construct level metrics, also only for constructs belonging to the same process
